recipe/meta.yaml

```diff
@@ -1,32 +1,35 @@
-{% set version = "5.0.2" %}
+{% set name = "kombu" %}
+{% set version = "5.2.2" %}
+
 
 package:
-  name: kombu
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/k/kombu/kombu-{{ version }}.tar.gz
-  sha256: f4965fba0a4718d47d470beeb5d6446e3357a62402b16c510b6a2f251e05ac3c
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kombu-{{ version }}.tar.gz
+  sha256: 0f5d0763fb916808f617b886697b2be28e6bc35026f08e679697fc814b48a608
+  patches:
+    - fix_test_txt.patch
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
```

Comment:
Are we missing patch as a requirement here?

Reply:
Thanks. Fixed it